phone/view.py

```diff
@@ -0,0 +1,41 @@
+import controller
+def menu():
+    print('''Меню:
+        1. Открыть файл
+        2. Сохранить файл
+        3. Показать контакты
+        4. Создать контакт
+        5. Изменить контакт
+        6. Найти контакт
+        7. Удалить контакт
+        8. Выход''')
+    choise = int(input("Выберите пункт меню: "))
+    return choise
```

Comment:
Зачем вам эта переменная, если вы используете ее всего 1 раз.
```suggestion
    return int(input("Выберите пункт меню: "))
```